build/installers/installer-tests/pom.xml

```diff
@@ -1,56 +1,59 @@
 <?xml version="1.0"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>com.liferay.ide.build</groupId>
-    <artifactId>installers</artifactId>
-    <version>3.3.1-SNAPSHOT</version>
-  </parent>
-  <groupId>com.liferay.ide.installer.tests</groupId>
-  <artifactId>installer-tests</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <name>installer-tests</name>
-  <url>http://maven.apache.org</url>
-  <dependencies>
-<dependency>
-    <groupId>org.junit.platform</groupId>
-    <artifactId>junit-platform-launcher</artifactId>
-    <version>1.1.0</version>
-    
-</dependency>
-<dependency>
-    <groupId>org.junit.jupiter</groupId>
-    <artifactId>junit-jupiter-engine</artifactId>
-    <version>5.1.0</version>
-    
-</dependency>
-<dependency>
-    <groupId>org.junit.vintage</groupId>
-    <artifactId>junit-vintage-engine</artifactId>
-    <version>5.1.0</version>
-    
-</dependency>
-  </dependencies>
-  <build>
-    <plugins>
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.liferay.ide.build</groupId>
+		<artifactId>installers</artifactId>
+		<version>3.3.1-SNAPSHOT</version>
+	</parent>
+	<groupId>com.liferay.ide.installer.tests</groupId>
+	<artifactId>installer-tests</artifactId>
+	<version>1.0-SNAPSHOT</version>
+	<name>installer-tests</name>
+	<url>http://maven.apache.org</url>
+	<dependencies>
+		<dependency>
+			<groupId>org.junit.platform</groupId>
+			<artifactId>junit-platform-launcher</artifactId>
+			<version>1.1.0</version>
 
-        <plugin>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <version>2.19.1</version>
-            <dependencies>
-                <dependency>
-                    <groupId>org.junit.platform</groupId>
-                    <artifactId>junit-platform-surefire-provider</artifactId>
-                    <version>1.1.0</version>
-                </dependency>
-                <dependency>
-                    <groupId>org.junit.jupiter</groupId>
-                    <artifactId>junit-jupiter-engine</artifactId>
-                    <version>5.1.0</version>
-                </dependency>
-            </dependencies>
-        </plugin>
-    </plugins>
-</build>
-</project>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>5.1.0</version>
+
+		</dependency>
+		<dependency>
+			<groupId>org.junit.vintage</groupId>
+			<artifactId>junit-vintage-engine</artifactId>
```

Comment:
If we are writing tests using junit jupitor api but we have the vintage engine that means our tests will still be junit4 right?  Do we plan to update these tests using junit 5/jupitor features?